Gemfile

```diff
@@ -3,14 +3,17 @@
 source "https://rubygems.org"
 gemspec
 
+gem "factory_bot", "~> 6.3.0"
```

Comment:
🗒️ I had to pin at 6.3 because newer versions of FactoryBot do not support Ruby 2.7, which is still part of our CI.